Mathlib/Topology/Basic.lean

```diff
@@ -902,6 +914,14 @@ theorem nhds_basis_closeds (x : X) : (𝓝 x).HasBasis (fun s : Set X => x ∉ s
     compl_surjective.exists.trans <| by simp only [isOpen_compl_iff, mem_compl_iff]⟩
 #align nhds_basis_closeds nhds_basis_closeds
 
+@[simp]
+theorem lift'_interior_nhds (x : X) : (𝓝 x).lift' interior = 𝓝 x :=
```

Comment:
The closure version is [lift'_nhds_closure](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/Separation.html#lift'_nhds_closure) so I think it's better to swap `interior` and `nhds` in the name (and same for `nhdsSet` below).